api/alembic/versions/2b3755392ad8_percent_grass_curing.py

```diff
@@ -0,0 +1,41 @@
+"""Percent grass curing
+
+Revision ID: 2b3755392ad8
+Revises: 403586c146ae
+Create Date: 2024-02-01 16:45:05.914743
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from app.db.models.common import TZTimeStamp
+
+# revision identifiers, used by Alembic.
+revision = '2b3755392ad8'
+down_revision = '403586c146ae'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic ###
+    op.create_table('percent_grass_curing',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('station_code', sa.Integer(), nullable=False),
+    sa.Column('percent_grass_curing', sa.Float(), nullable=False),
+    sa.Column('for_date', TZTimeStamp(), nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    comment='Record containing information about percent grass curing from the CFWIS.'
+    )
+    op.create_index(op.f('ix_percent_grass_curing_for_date'), 'percent_grass_curing', ['for_date'], unique=False)
+    op.create_index(op.f('ix_percent_grass_curing_id'), 'percent_grass_curing', ['id'], unique=False)
+    op.create_index(op.f('ix_percent_grass_curing_station_code'), 'percent_grass_curing', ['station_code'], unique=False)
```

Comment:
Curiosity:
Do you guys always follow alembic's default of adding an index for each single column?

Reply:
This is a great question, especially since indices take up space and can result in a lot of bloat in a database. The creation of indices is driven by the underlying model definition. For example, I intentionally did not add an index to the grass curing field: `percent_grass_curing = Column(Float, nullable=False, index=False)` as I don't see a use case for this.

Indexes on the `id` field are pretty standard, the others are really up to the dev or required use cases. I anticipated a need for an index on the station code and for_date fields for the last bit of work we'll be doing to wire up grass curing so included those in this PR.

Reply:
Thank you, that all makes sense. I was thinking the index for my `grass_curing` column was probably unnecessary for the same reasons
Also wondering if you guys ever make use of multicolumn indexes (or have found them useful) for certain queries you know will be commonly used and filter on those multiple columns

Reply:
We tried using one to address a performance issue a while back but it wasn't enough to fix the issue we were having. There's probably a place for them, but I don't have much experience with their use.